<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WildFox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/wildfox/icon</normaloff>:/wildfox/icon</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forwardButton">
        <property name="text">
         <string>Forward</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reloadButton">
        <property name="text">
         <string>Reload</string>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addressBar"/>
      </item>
      <item>
       <widget class="QPushButton" name="addressBarGo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Page_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFull_Screen"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="actionShow_all_history"/>
    <addaction name="separator"/>
    <addaction name="actionRecently_closed_tabs"/>
    <addaction name="actionRecently_closed_Windows"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionBookmark_this_page"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_WildFox"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHistory"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewTab">
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>&amp;New Window</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave_Page_As">
   <property name="text">
    <string>Save Page As...</string>
   </property>
  </action>
  <action name="actionNavigateBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="actionNavigateForward">
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>Next Page</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionShow_all_history">
   <property name="text">
    <string>Show all history</string>
   </property>
  </action>
  <action name="actionRecently_closed_tabs">
   <property name="text">
    <string>Recently closed tabs</string>
   </property>
  </action>
  <action name="actionRecently_closed_Windows">
   <property name="text">
    <string>Recently closed windows</string>
   </property>
  </action>
  <action name="actionBookmark_this_page">
   <property name="text">
    <string>Bookmark this page</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout_WildFox">
   <property name="text">
    <string>About WildFox</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="toolTip">
    <string>close tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload page</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>stop</string>
   </property>
   <property name="toolTip">
    <string>Stop loading page.</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionGo_to_address_bar">
   <property name="text">
    <string>Go to address bar</string>
   </property>
   <property name="toolTip">
    <string>Go to Address Bar</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
